docs/src/algorithms/clipping.md

```diff
@@ -28,6 +28,30 @@ other = Box((0,1), (3,7))
 # clipped polygon
 clipped = clip(poly, other, SutherlandHodgmanClipping())
 
+viz(poly)
+viz!(other, color = :black, alpha = 0.2)
+viz!(boundary(clipped), color = :red, segmentsize = 3)
+Mke.current_figure()
+```
+
+## Weiler-Atherton
+
+```@docs
+WeilerAthertonClipping
+```
+
+```@example clipping
+# polygon to clip
+outer = Ring((8, 0), (4, 8), (2, 8), (-2, 0), (0, 0), (1, 2), (5, 2), (6, 0))
+inner = Ring((4, 4), (2, 4), (3, 6))
```

Comment:
the inner ring should have CW orientation in the PolyArea constructor

Reply:
To me, this looks CW.